doc/doap.xml

```diff
@@ -0,0 +1,718 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet href="doap.xsl" type="text/xsl"?>
+
+<!--
+SPDX-FileCopyrightText: 2022 Melvin Keskin <melvo@olomono.de>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+<rdf:RDF xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#'
+         xmlns='http://usefulinc.com/ns/doap#'
+         xmlns:xmpp='https://linkmauve.fr/ns/xmpp-doap#'
+         xmlns:schema='https://schema.org/'>
+  <Project>
+    <name>QXmpp</name>
+    <created>2009-01-25</created>
```

Comment:
```suggestion
    <created>2008-12-01</created>
```
as far as I know QXmpp was already started in dec 2008, in 2009 the svn repo was created

Reply:
Do you have any reference for the exact date?

Reply:
unfortunately not

Reply:
Where do you have that information from?

Reply:
2008-20xx was used in the copyright headers, I don't know where I read about december (but december 01 is probably a good estimate)

Reply:
OK.